### DISCUSSION TITLE
what is Image Format the 8bit, 1 Channel, Grayscale

### DISCUSSION THREAD

**Discussion by yunhyejoung** [ImageMagick/ImageMagick#3374] [Help]
What is Image Format the 8bit, 1 Channel, Grayscale in ImageMagick.

"GRAY", CharPixel etc...

**Comment by dlemstra** [comment]
Please stop creating discussions. You have created 5 now with the same "question". And we cannot help you with opencv because we have no experience with that API.

**Reply by yunhyejoung** [reply]
What is Image Format the 8bit, 1 Channel, Grayscale in ImageMagick.
"GRAY", CharPixel etc...

**Reply by dlemstra** [reply]
I know you are new here @yunhyejoung but creating new discussions with the same question will not get you answer quicker.

**Comment by snibgo** [comment]
It seems that you are calling an ImageMagick function "mgk.write" which needs an ImageMagick structure "Image", but you are giving it an openCV structure "opencvImage.data".
